src/events.njk

```diff
@@ -81,11 +81,12 @@ endif %}
         timeStyle: "long",
         dateStyle: "medium",
       });
-      node.innerHTML = newLocaleString;
+      node.textContent = newLocaleString;
```

Comment:
Even though it would be fine in this case either way, I generally try to use `textContent` by default, since `innerHTML` is potentially dangerous.